semantic_version/base.py

```diff
@@ -1256,15 +1262,21 @@ def parse(cls, expression):
                 subclauses = []
                 if cls.HYPHEN in group:
                     low, high = group.split(cls.HYPHEN, 2)
-                    subclauses = cls.parse_simple('>=' + low) + cls.parse_simple('<=' + high)
+                    subclauses = cls.parse_simple('>=' + low.strip()) + cls.parse_simple('<=' + high.strip())
 
                 else:
-                    blocks = group.split(' ')
+                    blocks = group.split()
```

Comment:
This now splits across multiple whitespace, but also allows for newlines (so it would allow `>0.1.2 \n<3.4.5`)
Would it be worth raising an error at the start if the expression contains newlines?

Reply:
IMHO this is not worth

Reply:
I assume you're referring to "not worth raising an error for newlines", as opposed to allowing multiple spaces between the subclauses?

Reply:
> I assume you're referring to "not worth raising an error for newlines"

Correct. How many times are there new lines across all the package.json anyway?

Reply:
Yeah I wouldn't expect it as an issue, just a slight nitpick of my own code where it's different to the current npm implementation.